app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'ip-camera';
+
+export default class IpCameraService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  // Public getter only properties.
```

Comment:
Yeah, I'd make just a strict rule to group getters, public and private methods and order them appropriately.